docs/api.rst

```diff
@@ -22,6 +22,8 @@ Basic Exploration
 
 .. automethod:: groclient.GroClient.search_for_entity
 
+.. automethod:: groclient.GroClient.get_ancestors
```

Comment:
ahhh, just realized that we used singular name for `get_descendant` :( 
should we just call it `get_ancestor` ? any preference?  (sorry I know i put `ancestors` in original task  )

Reply:
I also thought about this, I thing `get_ancestor` is better, since it's consistent with the initial naming